### DISCUSSION TITLE
Предложение по лаунчеру.

### DISCUSSION THREAD

**Discussion by Juchara23** [octothorp-team/public-dobrograd#921] [Предложения]
Моё предложение весьма простое. Сделать ссылки на пользовательское соглашение, политику конфиденциальности, правила, контакты. Иначе не понятно что ты скачиваешь, нет гарантии что у тебя не украдут личные данные с компьютера и тому подобное. Насчёт этого вроде объяснил Ваничке, надеюсь добавите. Потому что лично для меня падает репутация проекта.

**Comment by chelog** [comment]
Я не думаю, что наличие этого в лаунчере что-то изменит, однако в нем уже есть ссылка на сайт, где есть вся эта информация:

![](https://i.imgur.com/qCsSRzT.png)

![](https://i.imgur.com/7EoNuiO.png)
